### PR TITLE
Chore/uniswapv3 docs

### PR DESCRIPTION
**Pull request by thecoderpanda**
@getjiggy UniswapV3 documentation added.

### CODE REVIEW

File: docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/category.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "label": "UniswapV3 Dashboard",
+    "position": 0,
+    "link": {
+      "type": "generated-index",
+      "description": "UniswapV3 Dashboard Implementation using Powerloom protocol"
```
Comment (SwaroopH) [inline docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/category.json:6]: P should be capital in protocol

File: docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### UniswapV3 WETH Pairs
+Some tokens cannot be priced using the OffchainOracle. For these tokens, we look for a UniswapV3 WETH pair and retreive the current price of the token in terms of WETH. This is then converted to a human readable price and used for data point calculations
+
+### UniswapV3 Stable Pairs
+If no UniswapV3 WETH pair exists for a given token, we fallback to UniswapV3 Stable pairs. As above, we retreive the current price of the token and convert it to a human readable price for calculations. The stable coins supported are USDC, USDT, and DAI. 
+
+## Tick Retreival
+
+In order to calculate our Total Value Locked data point, we must retreive every populated tick in a given pool. We accomplish this by batching RPC state overload calls to a modified [1inch UniV3Helper](https://github.com/getjiggy/evm-helpers). Once we have retreived all populated ticks for a pool, we store this information in a redis cache. This functionality is implemented in the get_tick_info function:
```
Comment (SwaroopH) [inline docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/index.md:36]: @anomit this should be on powerloom namespace ideally? We can come up with a utils or better repo name while we are at it.

**Review by SwaroopH — CHANGES_REQUESTED** [review]
Left comments

File: docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/category.json (hunk 1 above)
Comment (getjiggy) [reply docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/category.json]: pushed update

File: docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/index.md (hunk 2 above)
Comment (SwaroopH) [reply docs/build-with-powerloom/use-cases/existing-implementations/uniswapv3-dashboard/index.md:36]: @getjiggy let's use this: https://github.com/PowerLoom/evm-helpers/tree/pooler

**Review by SwaroopH — APPROVED** [review]
LGTM

**Pull request closed without merging**